concepts/strings/about.md

```diff
@@ -17,6 +17,19 @@ escaped == verbatim;
 // => true
 ```
 
+At some point you might need to separate a string into multiple parts. This might be because you want to concatenate different parts from multiple strings or you 
+need to analyse a specific part more closely but don't need the rest. That happens mostly with application logs.
+
+The easiest way to do this is with the [Substring method][substring] 
+
+```csharp
+string sentence = "Frank chases the bus.";
+string name = sentence.Substring(0,5)
+// => "Frank"
+```
+
+If you need to split the string at a certain character you can use [Substring][substring] in combination with [IndexOf][indexof]. In recent C# versions also the [Split method][splitting] has become very powerful. Check this out as well. 
```

Comment:
Maybe we can expand on this (`IndexOf()` a bit more?

````suggestion
The [`IndexOf`() method][indexof] can be used to find the index of the first occurence of a `string` within a `string`, returning `-1` if the specified value could not be found:

```csharp
"continuous-integration".IndexOf("integration")
// => 11

"continuous-integration".IndexOf("deployment")
// => -1
```
````

I'd prefer not to mention the `Split()` method, as that returns an array which the student won't be familiar with a this point (for the same reason, I've used strings with `IndexOf`).